### PR TITLE
Add note about V2 Smart Proxy features api

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ This section will be updated prior to the next release.
 Global parameters now support the same types as smart variables and smart class parameters.
 As a part of this change searching by parameter value is no longer allowed. ([#4127](https://projects.theforeman.org/issues/4127))
 
+#### Smart Proxy Exposed Capabilities and Settings
+
+The Smart Proxy now provides a v2 features api allowing plugins to declare capabilities and expose settings from their configuration.
```
Comment (mmoll) [inline _includes/manuals/nightly/1.2_release_notes.md]: api -> API, perhaps?

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 2]
```diff
@@ -9,6 +9,12 @@ This section will be updated prior to the next release.
...
+The Foreman server has been updated to use this new api and store the declared capabitlities and settings.  These will be used in the
```
Comment (mmoll) [inline _includes/manuals/nightly/1.2_release_notes.md]: same here

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 3]
```diff
@@ -9,6 +9,12 @@ This section will be updated prior to the next release.
...
+future to enhance and simplify our interactions with the Smart Proxy in the future.
```
Comment (tbrisker) [inline _includes/manuals/nightly/1.2_release_notes.md]: in the future... in the future

Comment (tbrisker) [reply _includes/manuals/nightly/1.2_release_notes.md]: also, perhaps link to the redmine issue(s) or discourse discussion so people interested in more information can find it?

**Review by tbrisker — APPROVED** [review]
Thanks @jlsherrill, pushed with the requested changes to your branch so we can merge.

File: _includes/manuals/nightly/1.2_release_notes.md (hunk 3 above)
Comment (ekohl) [reply _includes/manuals/nightly/1.2_release_notes.md]: We can add a link to https://github.com/theforeman/theforeman.org/pull/1342 once it's finished and published.

**Pull request merged**